docs/source/nlu.rst

```diff
@@ -55,4 +55,16 @@ Below you can see all the imports that are used, only from rasa.
 Entity Extraction
 -----------------
 
-As of now only one implementation exists, the Rasa DIET classifier.
\ No newline at end of file
+As of now only one implementation exists, the Rasa DIET classifier.
+
+Satisfaction Classification
+---------------------------
+
+The current satisfaction classifier is a SVM model pre-trained on `english data <https://github.com/sunnweiwei/user-satisfaction-simulation>`_.
```

Comment:
Shouldn't the simulation bit be in UserSimCRS? Since DialogueKit is primarily a tool for building conversational agents in general? I don't believe we train it on past dialogs at the moment, we use the pre-trained model directly. It's used to filter/select more appropriate response (NLG).